src/DynamoApplications/StartupUtils.cs

```diff
@@ -411,6 +411,10 @@ private static bool PreloadASM(string asmPath, out string geometryFactoryPath, o
                 IsServiceMode = isServiceMode,
                 Preferences = PreferenceSettings.Instance
             };
+            config.AuthProvider = CLImode ? null : new Core.IDSDKManager();
```

Comment:
It looks like the alias version of Sandbox should never get here. They have their own MakeModel implementation and are not calling any of ours as far as I can tell.